.github/workflows/notify-on-force-push.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     name: Notify on any direct push to `main`
     if: >
       github.repository == 'nodejs/release-cloudflare-worker' && 
-      github.actor != 'github-merge-queue[bot]'
+      github.event.forced
```

Comment:
Shouldn't we alert any push that was done through a PR merge? But also with this :)

Reply:
I don't really think each pr merge should be an alert, the idea was to alert for direct pushes to the main branch w/o a pr